templates/statefulset.yaml

```diff
@@ -115,26 +115,44 @@ spec:
 
           cat "${NIFI_HOME}/conf/nifi.temp" > "${NIFI_HOME}/conf/nifi.properties"
 
+          /opt/nifi/nifi-toolkit-current/bin/tls-toolkit.sh standalone -n "${FQDN}" \
```

Comment:
Would we still want this change?

Reply:
This has been updated to something more appropriate